src/main/java/com/lorepo/icplayer/client/printable/PrintableContentParser.java

```diff
@@ -515,16 +543,40 @@ private List<String> generatePageHTMLs(List<Page> sourcePages) {
 			pages = sourcePages;
 		}
 		List<String> parsedPages = new ArrayList<String>();
-		for (Page page: pages) {
-			String result = "<div class='page";
-			result += page.isSplitInPrintBlocked() ? "'>" : " splittable'>";
-			result += parsePage(page, randomizeModules, showAnswers);
-			result += "</div>";
-			parsedPages.add(result);
+
+		if (isPrintableOrderSet) {
+			for (int i=0; i<pages.size(); i++) {
+				Page page = this.getPageInPrintableOrder(pages, i);

```

Comment:
Nie rozumiem dlaczego wprowadziłeś tu if'a. Skutkiem tej zmiany jest, że uzyskamy wydruk stron względem kolejności w dicie.
W zadaniu nie znalazłem informacji, że to oczekiwany efekt. Mam wrażanie, że tak nie powinno to działać, gdyż ta metoda otrzymuje dict'a (zbiór klucz wartość, która zazwyczaj nie utrzymuje tej samej kolejności).

Reply:
po konsultacji z Lea usunąłem sortowanie stron